radiography/src/main/java/radiography/ScannableView.kt

```diff
@@ -68,9 +70,25 @@ public sealed class ScannableView {
     public val semanticsConfigurations: List<SemanticsConfiguration>
       get() {
         return semanticsNodes.map { it.config }.ifEmpty { null }
+          // Before Compose 1.6, semantics could be held in a SemanticsModifier
           ?: modifiers
             .filterIsInstance<SemanticsModifier>()
             .map { it.semanticsConfiguration }
+            .ifEmpty { null }
+            // After Compose 1.6, semantics are held in nodes. While this usually gets represented
+            // by semanticsNodes, sometimes they instead are only found in SemanticsModifierNode,
+            // which we must then extract.
+          ?: modifiers
```

Comment:
I needed to add this to get a few tests to pass. In some cases, the semanticsNodes are not present for a layout node (and I'm not sure why). However, I noticed that they are instead found within a special type of modifier node that we can leverage.

I would love if someone more familiar with compose internals can validate this approach. This fixes all test cases, but I am not 100% confident that it is the best way to do so. Ideally I think the `computeLayoutInfos` would be able to identify the semanticsNodes for each LayoutNodeInfo instead of needing to do this